src/Components/Sidebar/index.jsx

```
@@ -1,862 +1,862 @@
+	Tooltip,
+	Typography,
+} from "@mui/material";
+import ThemeSwitch from "../ThemeSwitch";
+import Avatar from "../Avatar";
+import StarPrompt from "../StarPrompt";
+import LockSvg from "../../assets/icons/lock.svg?react";
+import UserSvg from "../../assets/icons/user.svg?react";
+import TeamSvg from "../../assets/icons/user-two.svg?react";
+import LogoutSvg from "../../assets/icons/logout.svg?react";
+import Support from "../../assets/icons/support.svg?react";
+import Account from "../../assets/icons/user-edit.svg?react";
```

ESLint

[error] 'Account' is defined but never used.

(no-unused-vars)